content/en/docs/release-oversight/disruption-testing/data-architecture.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Architecture Data Flow"
+description: A high level look at how the disruption historical data is gathered and updated.
+weight: 1
+---
+
+### Resources
+
+{{% alert title="⚠️ Note!" color="warning" %}}
+You'll need access to the appropriate groups to work with disruption data, please reach out to the TRT team for access.
+{{% /alert %}}
+
+- [Periodic Jobs](https://github.com/openshift/release/tree/master/ci-operator/jobs/openshift/release)
+- [BigQuery](https://console.cloud.google.com/bigquery?project=openshift-ci-data-analysis)
+- [DPCR Job Aggregation Configs](https://github.com/openshift/continuous-release-jobs/tree/master/config/clusters/dpcr/services/dpcr-ci-job-aggregation)
+- [Origin Synthetic Backend Tests](https://github.com/openshift/origin/tree/master/pkg/synthetictests/allowedbackenddisruption)
+
+## Disruption Data Architecture
+
+{{% alert color="info" %}}
+The below diagram presents a high level overview on how we use our `periodic jobs`, `job aggregation` and `BigQuery` to generate the disruption historical data.
+It does not cover how the tests themselves are run against a cluster.
+{{% /alert %}}
+
+### High Level Diagram
+
+{{< inlineSVG file="/static/disruption_test_diagram.svg" >}}
+
+### How The Data Flows
+
+1. `Disruption uploader` jobs are run in the DPCR cluster, the current configuration can be found in the [openshift/continuous-release-jobs](https://github.com/openshift/continuous-release-jobs/tree/master/config/clusters/dpcr/services/dpcr-ci-job-aggregation).
+
+1. We grab a list of the Jobs names that we should gather from the `Jobs` table in `BigQuery`
+
+1. When e2e tests are done the results are uploaded to `GCS` and the results can be viewed in the artifacts folder for a particular job run. We only pull disruption data for job names specified in the `Jobs` table.
```

Comment:
I'd break this one up, I'm thinking note that when openshift-tests finished it uploads GCS disruption data to the bucket and the filename format as you have it below. I'd move this to #2 in the list.

Break the "we only pull data for specific jobs" out, and maybe combine that with the "We grab a list of jobs names" item. 

Then we flow from samplers -> gcs -> disruption uploader and how it works.

Reply:
I like that, updated in the PR to this fork.